frappe/core/doctype/doc_history_temp/doc_history_temp.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
It is, by definition, temporary.

Since actually figuring out which fields have changed shouldn't happen right when the user saves (can take too long), the old and new versions of the doc are dumped (as JSON) into the doc_history_temp entry. A worker queue processes these entries on whichever interval we decide on, makes the appropriate history entries, and then deletes the doc_history_temp record.